docs/docs/tutorials/get-started/1-build-basic-chatbot.md

```diff
@@ -45,24 +54,45 @@ class State(TypedDict):
 
 graph_builder = StateGraph(State)
 ```
+:::
+
+:::js
+```typescript
+import { Annotation } from "@langchain/langgraph";
+import { StateGraph, START, END } from "@langchain/langgraph";
+import { BaseMessage } from "@langchain/core/messages";
+
+const State = Annotation.Root({
+  // Messages have the type "BaseMessage[]". The reducer function
+  // defines how this state key should be updated
+  // (in this case, it appends messages to the list, rather than overwriting them)
+  messages: Annotation<BaseMessage[]>({
+    reducer: (x, y) => x.concat(y),
+  }),
+});
+
+const graphBuilder = new StateGraph(State);
+```
+:::
```

Comment:
```ts
import { StateGraph, MessagesZodState, START } from "@langchain/langgraph";
import { z } from "zod";

const graph = new StateGraph(
  z.object({ messages: MessagesZodState.shape.messages }),
)
```

Reply:
Could you paraphrase the statement? I'm not sure how to interpret the "otherwise".

Is the first example correct? Does it work in Studio?